newjersey/navigator.business.nj.gov
# feat: [187440151] Elevator registration municipality search

## Description

Created a new context for Housing Municipality and loaded it through static props



### Ticket



[187440151](https://www.pivotaltracker.com/story/show/187440151)

### Approach



### Steps to Test


Create a business that is not home-based and does have elevator devices (non-essential question in profile)
Go to Elevator Registration Task
Search for building using the dropdown and the values in the testing sheet to see results

### Notes



## Code author checklist
